fr_fr.lang

```diff
@@ -180,7 +179,7 @@ gui.xaero_delete_set=Supprimer Liste
 gui.xaero_delete_set_message=Etes vous sur de vouloir supprimer la liste actuelle ?
 gui.xaero_delete_set_message2=La liste sera supprimée du monde actuel.
 gui.xaero_toggle_waypoints=Afficher Les Points
-gui.xaero_dots_scale=Taille Des Points Des Entités (If it don't fit, you can remove the two "Des")
-gui.xaero_entity_info=Information Des Entités (Same here)
+gui.xaero_dots_size=Taille Des Points Des Entités
+gui.xaero_entity_info=Information Des Entités
```

Comment:
Holy. It definitely didn't fit lol.

Reply:
With the text in parentheses, no doubt ;D